src/sdl2.re

```diff
@@ -579,6 +579,10 @@ module MessageBox = {
     "resdl_SDL_ShowSimpleMessageBox";
 };
 
+module Timekeeping = {
+    external getTicks: unit => int = "resdl_SDL_GetTicks";
+}
```

Comment:
Apparently there's no Hygiene Check set up for the CI here, but this formatting is off. You should run `esy format`.

Reply:
Fixed :)